content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -85,6 +85,7 @@ sudo apt-cache madison kubeadm
 ```shell
 # Find the latest {{< skew currentVersion >}} version in the list.
 # It should look like {{< skew currentVersion >}}.x-*, where x is the latest patch.
+sudo yum clean all --disablerepo="*" --enablerepo=kubernetes # Make sure the YUM cache of the kubernetes repo is cleaned.
```

Comment:
I'd like SIG Cluster Lifecycle to confirm that this is what we expect to document.

A blanket recommendation may have cost implications for SIG K8s Infra, too (people pop this sort of thing in cron jobs).